### PR TITLE
Correct feeds

### PR DESCRIPTION
**Pull request by ldidry**
Small corrections to have valid feeds (validity checked from [W3C validator](https://validator.w3.org/feed/))

### CODE REVIEW

File: app/Http/Controllers/RssController.php [hunk 1]
```diff
@@ -60,9 +60,19 @@ private function feedAddItem(&$feed, $incident)
...
             Setting::get('app_name'),
-            Setting::get('app_domain'),
+            $this->canonicalizeUrl(Setting::get('app_domain')).'#'.$incident->id,
             $incident->created_at->toRssString(),
             $incident->message
         );
     }
+
+    /**
+     * Add a / at the end of an URL to in order to be W3C compliant
```
Comment (GrahamCampbell) [inline app/Http/Controllers/RssController.php]: Missing fullstop.

File: app/Http/Controllers/AtomController.php [hunk 2]
```diff
@@ -60,9 +60,19 @@ private function feedAddItem(&$feed, $incident)
...
             Setting::get('app_name'),
-            Setting::get('app_domain'),
+            $this->canonicalizeUrl(Setting::get('app_domain')).'#'.$incident->id,
             $incident->created_at->toAtomString(),
             $incident->message
         );
     }
+
+    /**
+     * Add a / at the end of an URL to in order to be W3C compliant
```
Comment (GrahamCampbell) [inline app/Http/Controllers/AtomController.php]: Missing fullstop.

File: app/Http/Controllers/AtomController.php [hunk 3]
```diff
@@ -60,9 +60,19 @@ private function feedAddItem(&$feed, $incident)
...
+     *
+     * @param string $url
+     */
```
Comment (GrahamCampbell) [inline app/Http/Controllers/AtomController.php]: You must include a return annotation.

File: app/Http/Controllers/RssController.php [hunk 4]
```diff
@@ -60,9 +60,19 @@ private function feedAddItem(&$feed, $incident)
...
+     *
+     * @param string $url
+     */
```
Comment (GrahamCampbell) [inline app/Http/Controllers/RssController.php]: You must include a return annotation.

File: app/Http/Controllers/AtomController.php [hunk 5]
```diff
@@ -60,9 +60,19 @@ private function feedAddItem(&$feed, $incident)
...
+    private function canonicalizeUrl($url)
+    {
+        return preg_replace('/([^\/])$/', '$1/', $url);
```
Comment (GrahamCampbell) [inline app/Http/Controllers/AtomController.php]: This code should not be duplicated between controllers.

Comment (ldidry) [inline app/Http/Controllers/AtomController.php]: I thought so. How would you want to do it? An other class and these two classes inheriting from it?

Comment (GrahamCampbell) [inline app/Http/Controllers/AtomController.php]: A trait?

Comment (ldidry) [inline app/Http/Controllers/AtomController.php]: Where should I put it? Creating a directory Traits in `app/Http`? I'm not a PHP guy, I don't know the conventions.

Comment (GrahamCampbell) [inline app/Http/Controllers/AtomController.php]: TBH, the best thing here would probably be to just make a utilities class and stick this as a static method.

Comment (GrahamCampbell) [inline app/Http/Controllers/AtomController.php]: Actually, no...

Comment (GrahamCampbell) [inline app/Http/Controllers/AtomController.php]: Use the macro method on laravel's Str class. :)

**Comment by GrahamCampbell** [comment]
I'll just make the change. It'll be quicker. :)

**Comment by GrahamCampbell** [comment]
Thank you.

**Comment by ldidry** [comment]
Thank you @GrahamCampbell !

**Pull request merged**